internal/integration/fixtures/extra-volume-mounts.yaml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - label: ":wave:"
+    agents:
+      queue: {{.queue}}
+    env:
+      BUILDKITE_SHELL: /bin/sh -e -c
+      BUILDKITE_GIT_CLONE_MIRROR_FLAGS: --mirror
+      BUILDKITE_GIT_MIRRORS_PATH: /tmp/buildkite-git-mirrors
+    plugins:
+      - kubernetes:
+          podSpec:
+            containers:
+              - image: alpine:latest
+                command: [bash]
```

Comment:
```suggestion
                command: [ash]
```

Reply:
Bash isn't present on alpine, which might be why this integration test is failing.